recipes-bsp/uefi/edk2_git.bb

```diff
@@ -60,6 +60,13 @@ do_compile() {
 
 do_deploy() {
     # Placeholder to be implemented in machine specific recipe
+
+    if [ -e ${EDK2_DIR}/atf/build/${UEFIMACHINE}/release/bl1.bin ] ; then 
+    	install -D -p -m0644 ${EDK2_DIR}/atf/build/${UEFIMACHINE}/release/bl1.bin ${DEPLOYDIR}/bl1.bin
+    fi
+    if [ -e ${EDK2_DIR}/atf/build/${UEFIMACHINE}/release/fip.bin ] ; then 
```

Comment:
If we install fip.bin in the generic edk2 recipe, we should clean up hikey specific recipe and remove the related install line.